### PR TITLE
GUI tweaks

### PR DESCRIPTION
**Pull request by jrslv**
This branch fixes several issues in GUI and adds support email link to GUI and documentation.

The email address is mesos-es@container-solutions.com. If we would like to invite someone who contacts us to Slack, we can do that (https://containersolutions.slack.com/admin/invites), but Slack has a funny limit of Single-Channel Guests: 5 times the number of paying users. That's why we decided to use email as the default means of contact for now.

Emails sent to this group are delivered to Frank, Lukasz, Phil, Martin, James, Pini and Jaroslav.

### CODE REVIEW

**Comment by jrslv** [comment]
Ready for review and merge.

File: scheduler/src/main/resources/public/partials/cluster.html [hunk 1]
```diff
@@ -27,11 +27,11 @@ <h3 class="font-red-haze">{{configuration.NumberOfHwNodes}}</h3>
             <!--</div>-->
         </div>
     </div>
-    <div class="col-md-3">
+    <div class="col-md-4">
         <div class="cell dashboard-stat">
             <div class="display">
                 <div class="number">
-                    <h3 class="font-red-haze">{{configuration.Mem}}{{configuration.MemUnits}}</h3>
+                    <h3 class="font-red-haze">{{configuration.Mem}}MB</h3>
```
Comment (philwinder) [inline scheduler/src/main/resources/public/partials/cluster.html]: Ah, yeah, sorry I broke this. But there is no related Mesos concept of units, do I deleted it.

**Comment by philwinder** [comment]
All looks good to me @jrslv Are you ready to merge in?

**Comment by jrslv** [comment]
Yes @philwinder, this batch of changes is ready for merge. No worries about the units.

**Pull request merged**